### PR TITLE
Compression documentation

### PR DESCRIPTION
**Pull request by huv1k**
This PR adds documentation about `compression`

### CODE REVIEW

File: packages/next/README.md [hunk 1]
```diff
@@ -2142,6 +2142,18 @@ Note: `NODE_ENV` is properly configured by the `next` subcommands, if absent, to
 
 Note: we recommend putting `.next`, or your [custom dist folder](https://github.com/zeit/next.js#custom-configuration), in `.gitignore` or `.npmignore`. Otherwise, use `files` or `now.files` to opt-into a whitelist of files you want to deploy, excluding `.next` or your custom dist folder.
 
+### Compression
+Next.js provides [gzip](https://tools.ietf.org/html/rfc6713#section-3) compression to speed up sending content.
```
Comment (timneutkens) [inline packages/next/README.md]:
```suggestion
Next.js provides [gzip](https://tools.ietf.org/html/rfc6713#section-3) compression to compress rendered content and static files.
```

File: packages/next/README.md [hunk 2]
```diff
@@ -2142,6 +2142,18 @@ Note: `NODE_ENV` is properly configured by the `next` subcommands, if absent, to
...
+
+To disable **compression** in Next.js, set the `compression` to `false` in `next.config.js`:
```
Comment (timneutkens) [inline packages/next/README.md]:
```suggestion
In general you will want to enable compression on a HTTP proxy like nginx. 

To disable compression in Next.js, set the `compression` to `false` in `next.config.js`:
```

Comment (kachkaev) [reply packages/next/README.md]: > In general you will want to enable compression on a HTTP proxy like nginx.

👍 I'd also add a small statement explaining why (in high-load apps, we want to offload compression from a Node.js process to improve its performance).

**Comment by ijjk** [comment]
## Stats from current PR

<details>
<summary>Click to expand stats</summary>

| | zeit/next.js canary | huv1k/next.js compression-docs | Change |
| - | - | - | - |
| Build Duration | 14.1s | 13.9s | -262ms |
| `node_modules` Size | 43.6 MB | 43.6 MB | ⚠️  +317 B |
| Total Bundle (main, webpack, commons) Size | 206 kB | 206 kB | ✓ |
| Total Bundle (main, webpack, commons) gzip Size | 67.8 kB | 67.8 kB | ✓ |
| Client `_app` Size | 2.39 kB | 2.39 kB | ✓ |
| Client `_app` gzip Size | 1.08 kB | 1.08 kB | ✓ |
| Client `_error` Size | 8.22 kB | 8.22 kB | ✓ |
| Client `_error` gzip Size | 3.16 kB | 3.16 kB | ✓ |
| Client `pages/index` Size | 343 B | 343 B | ✓ |
| Client `pages/index` gzip Size | 246 B | 246 B | ✓ |
| Client `pages/link` Size | 4.08 kB | 4.08 kB | ✓ |
| Client `pages/link` gzip Size | 1.8 kB | 1.8 kB | ✓ |
| Client `pages/routerDirect` Size | 423 B | 423 B | ✓ |
| Client `pages/routerDirect` gzip Size | 306 B | 306 B | ✓ |
| Client `pages/withRouter` Size | 435 B | 435 B | ✓ |
| Client `pages/withRouter` gzip Size | 301 B | 301 B | ✓ |
| Client `main` Size | 15.4 kB | 15.4 kB | ✓ |
| Client `main` gzip Size | 5.35 kB | 5.35 kB | ✓ |
| Client `commons` Size | 188 kB | 188 kB | ✓ |
| Client `commons` gzip Size | 61.1 kB | 61.1 kB | ✓ |
| Client `webpack` Size | 1.53 kB | 1.53 kB | ✓ |
| Client `webpack` gzip Size | 778 B | 778 B | ✓ |
| Base Rendered Size | 1.35 kB | 1.35 kB | ✓ |
| Build Dir Size | 702 kB | 702 kB | ✓ |

</details>


<details>
<summary>Click to expand serverless stats</summary>

| | zeit/next.js canary | huv1k/next.js compression-docs | Change |
| - | - | - | - |
| Build Duration | 15.2s | 15.1s | -84ms |
| `node_modules` Size | 43.6 MB | 43.6 MB | ⚠️  +317 B |
| Total Bundle (main, webpack, commons) Size | 206 kB | 206 kB | ✓ |
| Total Bundle (main, webpack, commons) gzip Size | 67.8 kB | 67.8 kB | ✓ |
| Client `_app` Size | 2.39 kB | 2.39 kB | ✓ |
| Client `_app` gzip Size | 1.08 kB | 1.08 kB | ✓ |
| Client `_error` Size | 8.22 kB | 8.22 kB | ✓ |
| Client `_error` gzip Size | 3.16 kB | 3.16 kB | ✓ |
| Client `pages/index` Size | 343 B | 343 B | ✓ |
| Client `pages/index` gzip Size | 246 B | 246 B | ✓ |
| Client `pages/link` Size | 4.08 kB | 4.08 kB | ✓ |
| Client `pages/link` gzip Size | 1.8 kB | 1.8 kB | ✓ |
| Client `pages/routerDirect` Size | 423 B | 423 B | ✓ |
| Client `pages/routerDirect` gzip Size | 306 B | 306 B | ✓ |
| Client `pages/withRouter` Size | 435 B | 435 B | ✓ |
| Client `pages/withRouter` gzip Size | 301 B | 301 B | ✓ |
| Client `main` Size | 15.4 kB | 15.4 kB | ✓ |
| Client `main` gzip Size | 5.35 kB | 5.35 kB | ✓ |
| Client `commons` Size | 188 kB | 188 kB | ✓ |
| Client `commons` gzip Size | 61.1 kB | 61.1 kB | ✓ |
| Client `webpack` Size | 1.53 kB | 1.53 kB | ✓ |
| Client `webpack` gzip Size | 778 B | 778 B | ✓ |
| Serverless `pages/link` Size | 251 kB | 251 kB | ✓ |
| Serverless `pages/link` gzip Size | 67.8 kB | 67.8 kB | ⚠️  +1 B |
| Serverless `pages/index` Size | 244 kB | 244 kB | ✓ |
| Serverless `pages/index` gzip Size | 65.6 kB | 65.6 kB | ⚠️  +1 B |
| Serverless `pages/_error` Size | 243 kB | 243 kB | ✓ |
| Serverless `pages/_error` gzip Size | 65.4 kB | 65.4 kB | ✓ |
| Serverless `pages/routerDirect` Size | 244 kB | 244 kB | ✓ |
| Serverless `pages/routerDirect` gzip Size | 65.6 kB | 65.6 kB | ✓ |
| Serverless `pages/withRouter` Size | 244 kB | 244 kB | ✓ |
| Serverless `pages/withRouter` gzip Size | 65.7 kB | 65.7 kB | ⚠️  +1 B |
| Build Dir Size | 1.89 MB | 1.89 MB | ✓ |

</details>

**Comment by ijjk** [comment]
## Stats from current PR

<details>
<summary>Click to expand stats</summary>

| | zeit/next.js canary | huv1k/next.js compression-docs | Change |
| - | - | - | - |
| Build Duration | 14s | 13.8s | -141ms |
| `node_modules` Size | 43.6 MB | 43.6 MB | ⚠️  +581 B |
| Total Bundle (main, webpack, commons) Size | 206 kB | 206 kB | ✓ |
| Total Bundle (main, webpack, commons) gzip Size | 67.8 kB | 67.8 kB | ⚠️  +1 B |
| Client `_app` Size | 2.39 kB | 2.39 kB | ✓ |
| Client `_app` gzip Size | 1.08 kB | 1.08 kB | ⚠️  +1 B |
| Client `_error` Size | 8.22 kB | 8.22 kB | ✓ |
| Client `_error` gzip Size | 3.16 kB | 3.16 kB | ✓ |
| Client `pages/index` Size | 343 B | 343 B | ✓ |
| Client `pages/index` gzip Size | 246 B | 246 B | ✓ |
| Client `pages/link` Size | 4.08 kB | 4.08 kB | ✓ |
| Client `pages/link` gzip Size | 1.8 kB | 1.8 kB | ✓ |
| Client `pages/routerDirect` Size | 423 B | 423 B | ✓ |
| Client `pages/routerDirect` gzip Size | 306 B | 306 B | ✓ |
| Client `pages/withRouter` Size | 435 B | 435 B | ✓ |
| Client `pages/withRouter` gzip Size | 300 B | 301 B | ⚠️  +1 B |
| Client `main` Size | 15.4 kB | 15.4 kB | ✓ |
| Client `main` gzip Size | 5.35 kB | 5.35 kB | ✓ |
| Client `commons` Size | 188 kB | 188 kB | ✓ |
| Client `commons` gzip Size | 61.1 kB | 61.1 kB | ✓ |
| Client `webpack` Size | 1.53 kB | 1.53 kB | ✓ |
| Client `webpack` gzip Size | 778 B | 778 B | ✓ |
| Base Rendered Size | 1.35 kB | 1.35 kB | ✓ |
| Build Dir Size | 702 kB | 702 kB | ✓ |

</details>


<details>
<summary>Click to expand serverless stats</summary>

| | zeit/next.js canary | huv1k/next.js compression-docs | Change |
| - | - | - | - |
| Build Duration | 15.3s | 15.2s | -91ms |
| `node_modules` Size | 43.6 MB | 43.6 MB | ⚠️  +581 B |
| Total Bundle (main, webpack, commons) Size | 206 kB | 206 kB | ✓ |
| Total Bundle (main, webpack, commons) gzip Size | 67.8 kB | 67.8 kB | -1 B |
| Client `_app` Size | 2.39 kB | 2.39 kB | ✓ |
| Client `_app` gzip Size | 1.08 kB | 1.08 kB | -1 B |
| Client `_error` Size | 8.22 kB | 8.22 kB | ✓ |
| Client `_error` gzip Size | 3.16 kB | 3.16 kB | ✓ |
| Client `pages/index` Size | 343 B | 343 B | ✓ |
| Client `pages/index` gzip Size | 246 B | 246 B | ✓ |
| Client `pages/link` Size | 4.08 kB | 4.08 kB | ✓ |
| Client `pages/link` gzip Size | 1.8 kB | 1.8 kB | ✓ |
| Client `pages/routerDirect` Size | 423 B | 423 B | ✓ |
| Client `pages/routerDirect` gzip Size | 306 B | 306 B | ✓ |
| Client `pages/withRouter` Size | 435 B | 435 B | ✓ |
| Client `pages/withRouter` gzip Size | 301 B | 300 B | -1 B |
| Client `main` Size | 15.4 kB | 15.4 kB | ✓ |
| Client `main` gzip Size | 5.35 kB | 5.35 kB | ✓ |
| Client `commons` Size | 188 kB | 188 kB | ✓ |
| Client `commons` gzip Size | 61.1 kB | 61.1 kB | ✓ |
| Client `webpack` Size | 1.53 kB | 1.53 kB | ✓ |
| Client `webpack` gzip Size | 778 B | 778 B | ✓ |
| Serverless `pages/link` Size | 251 kB | 251 kB | ✓ |
| Serverless `pages/link` gzip Size | 67.8 kB | 67.8 kB | ✓ |
| Serverless `pages/index` Size | 244 kB | 244 kB | ✓ |
| Serverless `pages/index` gzip Size | 65.6 kB | 65.6 kB | -1 B |
| Serverless `pages/_error` Size | 243 kB | 243 kB | ✓ |
| Serverless `pages/_error` gzip Size | 65.4 kB | 65.4 kB | ✓ |
| Serverless `pages/routerDirect` Size | 244 kB | 244 kB | ✓ |
| Serverless `pages/routerDirect` gzip Size | 65.6 kB | 65.6 kB | -1 B |
| Serverless `pages/withRouter` Size | 244 kB | 244 kB | ✓ |
| Serverless `pages/withRouter` gzip Size | 65.7 kB | 65.7 kB | ✓ |
| Build Dir Size | 1.89 MB | 1.89 MB | ✓ |

</details>

File: packages/next/README.md [hunk 3]
```diff
@@ -2142,6 +2145,18 @@ Note: `NODE_ENV` is properly configured by the `next` subcommands, if absent, to
 
 Note: we recommend putting `.next`, or your [custom dist folder](https://github.com/zeit/next.js#custom-configuration), in `.gitignore` or `.npmignore`. Otherwise, use `files` or `now.files` to opt-into a whitelist of files you want to deploy, excluding `.next` or your custom dist folder.
 
+### Compression
+Next.js provides [gzip](https://tools.ietf.org/html/rfc6713#section-3) compression to compress rendered content and static files. Compression only works with `server` target. In general you will want to enable compression on a HTTP proxy like nginx, to offload load from `Node.js` process.
```
Comment (lfades) [inline packages/next/README.md]:
```suggestion
Next.js provides [gzip](https://tools.ietf.org/html/rfc6713#section-3) compression to compress rendered content and static files. Compression only works with the `server` target. In general you will want to enable compression on a HTTP proxy like [nginx](https://www.nginx.com/), to offload load from the `Node.js` process.  
```

File: packages/next/README.md [hunk 4]
```diff
@@ -2142,6 +2145,18 @@ Note: `NODE_ENV` is properly configured by the `next` subcommands, if absent, to
 
 Note: we recommend putting `.next`, or your [custom dist folder](https://github.com/zeit/next.js#custom-configuration), in `.gitignore` or `.npmignore`. Otherwise, use `files` or `now.files` to opt-into a whitelist of files you want to deploy, excluding `.next` or your custom dist folder.
 
+### Compression
+Next.js provides [gzip](https://tools.ietf.org/html/rfc6713#section-3) compression to compress rendered content and static files. Compression only works with `server` target. In general you will want to enable compression on a HTTP proxy like nginx, to offload load from `Node.js` process.  
+
+To disable **compression** in Next.js, set the `compression` to `false` in `next.config.js`:
```
Comment (lfades) [inline packages/next/README.md]:
```suggestion
To disable **compression** in Next.js, set `compression` as `false` in `next.config.js`:
```

**Comment by ijjk** [comment]
## Stats from current PR

<details>
<summary>Click to expand stats</summary>

| | zeit/next.js canary | huv1k/next.js compression-docs | Change |
| - | - | - | - |
| Build Duration | 13.7s | 13.7s | ⚠️  +29ms |
| `node_modules` Size | 43.6 MB | 43.6 MB | -3.02 kB |
| Total Bundle (main, webpack, commons) Size | 206 kB | 206 kB | ✓ |
| Total Bundle (main, webpack, commons) gzip Size | 67.8 kB | 67.8 kB | ✓ |
| Client `_app` Size | 2.39 kB | 2.39 kB | ✓ |
| Client `_app` gzip Size | 1.08 kB | 1.08 kB | ✓ |
| Client `_error` Size | 8.22 kB | 8.22 kB | ✓ |
| Client `_error` gzip Size | 3.16 kB | 3.16 kB | ✓ |
| Client `pages/index` Size | 343 B | 343 B | ✓ |
| Client `pages/index` gzip Size | 246 B | 246 B | ✓ |
| Client `pages/link` Size | 4.08 kB | 4.08 kB | ✓ |
| Client `pages/link` gzip Size | 1.8 kB | 1.8 kB | ✓ |
| Client `pages/routerDirect` Size | 423 B | 423 B | ✓ |
| Client `pages/routerDirect` gzip Size | 306 B | 306 B | ✓ |
| Client `pages/withRouter` Size | 435 B | 435 B | ✓ |
| Client `pages/withRouter` gzip Size | 301 B | 301 B | ✓ |
| Client `main` Size | 15.4 kB | 15.4 kB | ✓ |
| Client `main` gzip Size | 5.35 kB | 5.35 kB | ✓ |
| Client `commons` Size | 188 kB | 188 kB | ✓ |
| Client `commons` gzip Size | 61.1 kB | 61.1 kB | ✓ |
| Client `webpack` Size | 1.53 kB | 1.53 kB | ✓ |
| Client `webpack` gzip Size | 778 B | 778 B | ✓ |
| Base Rendered Size | 1.35 kB | 1.35 kB | ✓ |
| Build Dir Size | 702 kB | 702 kB | ✓ |

</details>


<details>
<summary>Click to expand serverless stats</summary>

| | zeit/next.js canary | huv1k/next.js compression-docs | Change |
| - | - | - | - |
| Build Duration | 15s | 14.7s | -321ms |
| `node_modules` Size | 43.6 MB | 43.6 MB | -3.02 kB |
| Total Bundle (main, webpack, commons) Size | 206 kB | 206 kB | ✓ |
| Total Bundle (main, webpack, commons) gzip Size | 67.8 kB | 67.8 kB | ✓ |
| Client `_app` Size | 2.39 kB | 2.39 kB | ✓ |
| Client `_app` gzip Size | 1.08 kB | 1.08 kB | ✓ |
| Client `_error` Size | 8.22 kB | 8.22 kB | ✓ |
| Client `_error` gzip Size | 3.16 kB | 3.16 kB | ✓ |
| Client `pages/index` Size | 343 B | 343 B | ✓ |
| Client `pages/index` gzip Size | 246 B | 246 B | ✓ |
| Client `pages/link` Size | 4.08 kB | 4.08 kB | ✓ |
| Client `pages/link` gzip Size | 1.8 kB | 1.8 kB | ✓ |
| Client `pages/routerDirect` Size | 423 B | 423 B | ✓ |
| Client `pages/routerDirect` gzip Size | 306 B | 306 B | ✓ |
| Client `pages/withRouter` Size | 435 B | 435 B | ✓ |
| Client `pages/withRouter` gzip Size | 301 B | 301 B | ✓ |
| Client `main` Size | 15.4 kB | 15.4 kB | ✓ |
| Client `main` gzip Size | 5.35 kB | 5.35 kB | ✓ |
| Client `commons` Size | 188 kB | 188 kB | ✓ |
| Client `commons` gzip Size | 61.1 kB | 61.1 kB | ✓ |
| Client `webpack` Size | 1.53 kB | 1.53 kB | ✓ |
| Client `webpack` gzip Size | 778 B | 778 B | ✓ |
| Serverless `pages/link` Size | 251 kB | 251 kB | ✓ |
| Serverless `pages/link` gzip Size | 67.8 kB | 67.8 kB | ✓ |
| Serverless `pages/index` Size | 244 kB | 244 kB | ✓ |
| Serverless `pages/index` gzip Size | 65.6 kB | 65.6 kB | ✓ |
| Serverless `pages/_error` Size | 243 kB | 243 kB | ✓ |
| Serverless `pages/_error` gzip Size | 65.4 kB | 65.4 kB | ✓ |
| Serverless `pages/routerDirect` Size | 244 kB | 244 kB | ✓ |
| Serverless `pages/routerDirect` gzip Size | 65.6 kB | 65.6 kB | ⚠️  +1 B |
| Serverless `pages/withRouter` Size | 244 kB | 244 kB | ✓ |
| Serverless `pages/withRouter` gzip Size | 65.7 kB | 65.7 kB | ✓ |
| Build Dir Size | 1.89 MB | 1.89 MB | ✓ |

</details>

**Comment by ijjk** [comment]
## Stats from current PR

<details>
<summary>Click to expand stats</summary>

| | zeit/next.js canary | huv1k/next.js compression-docs | Change |
| - | - | - | - |
| Build Duration | 13.2s | 12.7s | -444ms |
| `node_modules` Size | 43.6 MB | 43.6 MB | -3.02 kB |
| Total Bundle (main, webpack, commons) Size | 206 kB | 206 kB | ✓ |
| Total Bundle (main, webpack, commons) gzip Size | 67.8 kB | 67.8 kB | ✓ |
| Client `_app` Size | 2.39 kB | 2.39 kB | ✓ |
| Client `_app` gzip Size | 1.08 kB | 1.08 kB | ✓ |
| Client `_error` Size | 8.22 kB | 8.22 kB | ✓ |
| Client `_error` gzip Size | 3.16 kB | 3.16 kB | ✓ |
| Client `pages/index` Size | 343 B | 343 B | ✓ |
| Client `pages/index` gzip Size | 246 B | 246 B | ✓ |
| Client `pages/link` Size | 4.08 kB | 4.08 kB | ✓ |
| Client `pages/link` gzip Size | 1.8 kB | 1.8 kB | ✓ |
| Client `pages/routerDirect` Size | 423 B | 423 B | ✓ |
| Client `pages/routerDirect` gzip Size | 306 B | 306 B | ✓ |
| Client `pages/withRouter` Size | 435 B | 435 B | ✓ |
| Client `pages/withRouter` gzip Size | 301 B | 301 B | ✓ |
| Client `main` Size | 15.4 kB | 15.4 kB | ✓ |
| Client `main` gzip Size | 5.35 kB | 5.35 kB | ✓ |
| Client `commons` Size | 188 kB | 188 kB | ✓ |
| Client `commons` gzip Size | 61.1 kB | 61.1 kB | ✓ |
| Client `webpack` Size | 1.53 kB | 1.53 kB | ✓ |
| Client `webpack` gzip Size | 778 B | 778 B | ✓ |
| Base Rendered Size | 1.35 kB | 1.35 kB | ✓ |
| Build Dir Size | 702 kB | 702 kB | ✓ |

</details>


<details>
<summary>Click to expand serverless stats</summary>

| | zeit/next.js canary | huv1k/next.js compression-docs | Change |
| - | - | - | - |
| Build Duration | 13.9s | 13.8s | -89ms |
| `node_modules` Size | 43.6 MB | 43.6 MB | -3.02 kB |
| Total Bundle (main, webpack, commons) Size | 206 kB | 206 kB | ✓ |
| Total Bundle (main, webpack, commons) gzip Size | 67.8 kB | 67.8 kB | ✓ |
| Client `_app` Size | 2.39 kB | 2.39 kB | ✓ |
| Client `_app` gzip Size | 1.08 kB | 1.08 kB | ✓ |
| Client `_error` Size | 8.22 kB | 8.22 kB | ✓ |
| Client `_error` gzip Size | 3.16 kB | 3.16 kB | ✓ |
| Client `pages/index` Size | 343 B | 343 B | ✓ |
| Client `pages/index` gzip Size | 246 B | 246 B | ✓ |
| Client `pages/link` Size | 4.08 kB | 4.08 kB | ✓ |
| Client `pages/link` gzip Size | 1.8 kB | 1.8 kB | ✓ |
| Client `pages/routerDirect` Size | 423 B | 423 B | ✓ |
| Client `pages/routerDirect` gzip Size | 306 B | 306 B | ✓ |
| Client `pages/withRouter` Size | 435 B | 435 B | ✓ |
| Client `pages/withRouter` gzip Size | 301 B | 301 B | ✓ |
| Client `main` Size | 15.4 kB | 15.4 kB | ✓ |
| Client `main` gzip Size | 5.35 kB | 5.35 kB | ✓ |
| Client `commons` Size | 188 kB | 188 kB | ✓ |
| Client `commons` gzip Size | 61.1 kB | 61.1 kB | ✓ |
| Client `webpack` Size | 1.53 kB | 1.53 kB | ✓ |
| Client `webpack` gzip Size | 778 B | 778 B | ✓ |
| Serverless `pages/link` Size | 251 kB | 251 kB | ✓ |
| Serverless `pages/link` gzip Size | 67.8 kB | 67.8 kB | -2 B |
| Serverless `pages/index` Size | 244 kB | 244 kB | ✓ |
| Serverless `pages/index` gzip Size | 65.6 kB | 65.6 kB | -1 B |
| Serverless `pages/_error` Size | 243 kB | 243 kB | ✓ |
| Serverless `pages/_error` gzip Size | 65.4 kB | 65.4 kB | -1 B |
| Serverless `pages/routerDirect` Size | 244 kB | 244 kB | ✓ |
| Serverless `pages/routerDirect` gzip Size | 65.6 kB | 65.6 kB | ✓ |
| Serverless `pages/withRouter` Size | 244 kB | 244 kB | ✓ |
| Serverless `pages/withRouter` gzip Size | 65.7 kB | 65.7 kB | -1 B |
| Build Dir Size | 1.89 MB | 1.89 MB | ✓ |

</details>

**Comment by ijjk** [comment]
## Stats from current PR

<details>
<summary>Click to expand stats</summary>

| | zeit/next.js canary | huv1k/next.js compression-docs | Change |
| - | - | - | - |
| Build Duration | 12.9s | 13s | ⚠️  +144ms |
| `node_modules` Size | 43.6 MB | 43.6 MB | ⚠️  +611 B |
| Total Bundle (main, webpack, commons) Size | 206 kB | 206 kB | ✓ |
| Total Bundle (main, webpack, commons) gzip Size | 67.8 kB | 67.8 kB | -1 B |
| Client `_app` Size | 2.39 kB | 2.39 kB | ✓ |
| Client `_app` gzip Size | 1.08 kB | 1.08 kB | -1 B |
| Client `_error` Size | 8.22 kB | 8.22 kB | ✓ |
| Client `_error` gzip Size | 3.16 kB | 3.16 kB | ✓ |
| Client `pages/index` Size | 343 B | 343 B | ✓ |
| Client `pages/index` gzip Size | 246 B | 246 B | ✓ |
| Client `pages/link` Size | 4.08 kB | 4.08 kB | ✓ |
| Client `pages/link` gzip Size | 1.8 kB | 1.8 kB | ✓ |
| Client `pages/routerDirect` Size | 423 B | 423 B | ✓ |
| Client `pages/routerDirect` gzip Size | 306 B | 306 B | ✓ |
| Client `pages/withRouter` Size | 435 B | 435 B | ✓ |
| Client `pages/withRouter` gzip Size | 301 B | 300 B | -1 B |
| Client `main` Size | 15.4 kB | 15.4 kB | ✓ |
| Client `main` gzip Size | 5.35 kB | 5.35 kB | ✓ |
| Client `commons` Size | 188 kB | 188 kB | ✓ |
| Client `commons` gzip Size | 61.1 kB | 61.1 kB | ✓ |
| Client `webpack` Size | 1.53 kB | 1.53 kB | ✓ |
| Client `webpack` gzip Size | 778 B | 778 B | ✓ |
| Base Rendered Size | 1.35 kB | 1.35 kB | ✓ |
| Build Dir Size | 702 kB | 702 kB | ✓ |

</details>


<details>
<summary>Click to expand serverless stats</summary>

| | zeit/next.js canary | huv1k/next.js compression-docs | Change |
| - | - | - | - |
| Build Duration | 13.8s | 13.7s | -67ms |
| `node_modules` Size | 43.6 MB | 43.6 MB | ⚠️  +611 B |
| Total Bundle (main, webpack, commons) Size | 206 kB | 206 kB | ✓ |
| Total Bundle (main, webpack, commons) gzip Size | 67.8 kB | 67.8 kB | ✓ |
| Client `_app` Size | 2.39 kB | 2.39 kB | ✓ |
| Client `_app` gzip Size | 1.08 kB | 1.08 kB | ✓ |
| Client `_error` Size | 8.22 kB | 8.22 kB | ✓ |
| Client `_error` gzip Size | 3.16 kB | 3.16 kB | ✓ |
| Client `pages/index` Size | 343 B | 343 B | ✓ |
| Client `pages/index` gzip Size | 246 B | 246 B | ✓ |
| Client `pages/link` Size | 4.08 kB | 4.08 kB | ✓ |
| Client `pages/link` gzip Size | 1.8 kB | 1.8 kB | ✓ |
| Client `pages/routerDirect` Size | 423 B | 423 B | ✓ |
| Client `pages/routerDirect` gzip Size | 306 B | 306 B | ✓ |
| Client `pages/withRouter` Size | 435 B | 435 B | ✓ |
| Client `pages/withRouter` gzip Size | 301 B | 301 B | ✓ |
| Client `main` Size | 15.4 kB | 15.4 kB | ✓ |
| Client `main` gzip Size | 5.35 kB | 5.35 kB | ✓ |
| Client `commons` Size | 188 kB | 188 kB | ✓ |
| Client `commons` gzip Size | 61.1 kB | 61.1 kB | ✓ |
| Client `webpack` Size | 1.53 kB | 1.53 kB | ✓ |
| Client `webpack` gzip Size | 778 B | 778 B | ✓ |
| Serverless `pages/link` Size | 251 kB | 251 kB | ✓ |
| Serverless `pages/link` gzip Size | 67.8 kB | 67.8 kB | ✓ |
| Serverless `pages/index` Size | 244 kB | 244 kB | ✓ |
| Serverless `pages/index` gzip Size | 65.6 kB | 65.6 kB | ✓ |
| Serverless `pages/_error` Size | 243 kB | 243 kB | ✓ |
| Serverless `pages/_error` gzip Size | 65.4 kB | 65.4 kB | ✓ |
| Serverless `pages/routerDirect` Size | 244 kB | 244 kB | ✓ |
| Serverless `pages/routerDirect` gzip Size | 65.6 kB | 65.6 kB | ⚠️  +1 B |
| Serverless `pages/withRouter` Size | 244 kB | 244 kB | ✓ |
| Serverless `pages/withRouter` gzip Size | 65.7 kB | 65.7 kB | ✓ |
| Build Dir Size | 1.89 MB | 1.89 MB | ✓ |

</details>

**Review by timneutkens — APPROVED** [review]

**Comment by ijjk** [comment]
## Stats from current PR

<details>
<summary>Click to expand stats</summary>

| | zeit/next.js canary | huv1k/next.js compression-docs | Change |
| - | - | - | - |
| Build Duration | 12.2s | 11.9s | -232ms |
| `node_modules` Size | 43.6 MB | 43.6 MB | ⚠️  +8 B |
| Total Bundle (main, webpack, commons) Size | 206 kB | 206 kB | ✓ |
| Total Bundle (main, webpack, commons) gzip Size | 67.8 kB | 67.8 kB | ✓ |
| Client `_app` Size | 2.39 kB | 2.39 kB | ✓ |
| Client `_app` gzip Size | 1.08 kB | 1.08 kB | ✓ |
| Client `_error` Size | 8.22 kB | 8.22 kB | ✓ |
| Client `_error` gzip Size | 3.16 kB | 3.16 kB | ✓ |
| Client `pages/index` Size | 343 B | 343 B | ✓ |
| Client `pages/index` gzip Size | 246 B | 246 B | ✓ |
| Client `pages/link` Size | 4.08 kB | 4.08 kB | ✓ |
| Client `pages/link` gzip Size | 1.8 kB | 1.8 kB | ✓ |
| Client `pages/routerDirect` Size | 423 B | 423 B | ✓ |
| Client `pages/routerDirect` gzip Size | 306 B | 306 B | ✓ |
| Client `pages/withRouter` Size | 435 B | 435 B | ✓ |
| Client `pages/withRouter` gzip Size | 301 B | 301 B | ✓ |
| Client `main` Size | 15.4 kB | 15.4 kB | ✓ |
| Client `main` gzip Size | 5.35 kB | 5.35 kB | ✓ |
| Client `commons` Size | 188 kB | 188 kB | ✓ |
| Client `commons` gzip Size | 61.1 kB | 61.1 kB | ✓ |
| Client `webpack` Size | 1.53 kB | 1.53 kB | ✓ |
| Client `webpack` gzip Size | 778 B | 778 B | ✓ |
| Base Rendered Size | 1.35 kB | 1.35 kB | ✓ |
| Build Dir Size | 702 kB | 702 kB | ⚠️  +4 B |

</details>


<details>
<summary>Click to expand serverless stats</summary>

| | zeit/next.js canary | huv1k/next.js compression-docs | Change |
| - | - | - | - |
| Build Duration | 12.9s | 13s | ⚠️  +91ms |
| `node_modules` Size | 43.6 MB | 43.6 MB | ⚠️  +8 B |
| Total Bundle (main, webpack, commons) Size | 206 kB | 206 kB | ✓ |
| Total Bundle (main, webpack, commons) gzip Size | 67.8 kB | 67.8 kB | ✓ |
| Client `_app` Size | 2.39 kB | 2.39 kB | ✓ |
| Client `_app` gzip Size | 1.08 kB | 1.08 kB | ✓ |
| Client `_error` Size | 8.22 kB | 8.22 kB | ✓ |
| Client `_error` gzip Size | 3.16 kB | 3.16 kB | ✓ |
| Client `pages/index` Size | 343 B | 343 B | ✓ |
| Client `pages/index` gzip Size | 246 B | 246 B | ✓ |
| Client `pages/link` Size | 4.08 kB | 4.08 kB | ✓ |
| Client `pages/link` gzip Size | 1.8 kB | 1.8 kB | ✓ |
| Client `pages/routerDirect` Size | 423 B | 423 B | ✓ |
| Client `pages/routerDirect` gzip Size | 306 B | 306 B | ✓ |
| Client `pages/withRouter` Size | 435 B | 435 B | ✓ |
| Client `pages/withRouter` gzip Size | 301 B | 301 B | ✓ |
| Client `main` Size | 15.4 kB | 15.4 kB | ✓ |
| Client `main` gzip Size | 5.35 kB | 5.35 kB | ✓ |
| Client `commons` Size | 188 kB | 188 kB | ✓ |
| Client `commons` gzip Size | 61.1 kB | 61.1 kB | ✓ |
| Client `webpack` Size | 1.53 kB | 1.53 kB | ✓ |
| Client `webpack` gzip Size | 778 B | 778 B | ✓ |
| Serverless `pages/link` Size | 251 kB | 251 kB | ⚠️  +2 B |
| Serverless `pages/link` gzip Size | 67.8 kB | 67.8 kB | ⚠️  +2 B |
| Serverless `pages/index` Size | 244 kB | 244 kB | ⚠️  +2 B |
| Serverless `pages/index` gzip Size | 65.6 kB | 65.6 kB | ⚠️  +3 B |
| Serverless `pages/_error` Size | 243 kB | 243 kB | ⚠️  +2 B |
| Serverless `pages/_error` gzip Size | 65.4 kB | 65.4 kB | ⚠️  +3 B |
| Serverless `pages/routerDirect` Size | 244 kB | 244 kB | ⚠️  +2 B |
| Serverless `pages/routerDirect` gzip Size | 65.6 kB | 65.6 kB | ⚠️  +2 B |
| Serverless `pages/withRouter` Size | 244 kB | 244 kB | ⚠️  +2 B |
| Serverless `pages/withRouter` gzip Size | 65.7 kB | 65.7 kB | ⚠️  +2 B |
| Build Dir Size | 1.89 MB | 1.89 MB | ⚠️  +12 B |

</details>

**Pull request merged**